lib/addMethod/validateRESTInput.js

```diff
@@ -6,15 +6,15 @@ module.exports = function (methodName, config) {
 
 	// Ensure the minimum parameters have been passed
 	if (!methodName || !_.isString(methodName)) {
-		throw new Error('The first parameter passed to `addMethod` should be a string.');
+		throw new Error(`The first parameter passed to 'addMethod' should be a string. Operation: ${ _.snakeCase(methodName)}`);
```

Comment:
1. Threadneedle doesn't know about the concept of "Operation", that's a Falafel concept. So instead this should be "Method".
2. Isn't `methodName` already snakeCase coming from Falafel? In any case, Threadneedle needs to be more generic, so don't force any casing; use `methodName` as is.